.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
 
 script:
   - yarn lint
-  - travis_wait yarn test --ci --maxWorkers=2 --verbose --coverage && cat ./coverage/lcov.info | yarn run coveralls && rm -rf ./coverage # report coveralls status
+  - travis_wait yarn test --ci --maxWorkers=2 --coverage && cat ./coverage/lcov.info | yarn run coveralls && rm -rf ./coverage # report coveralls status
```

Comment:
Is the `--verbose` flag causing issues?

Reply:
yeah it was causing intermittent travis errors

Reply:
https://blog.m157q.tw/posts/2018/03/30/travis-ci-stdout-write-error-and-resource-temporarily-unavailable-workaround/